### PR TITLE
New InDesign cheat sheet

### PR DESCRIPTION
**Pull request by navneetzz**
What does your Instant Answer do?
My Instant answer provide shortcut keys used in adobe illustrator.

What is the data source for your Instant Answer? (Provide a link if possible)
Adobe help (https://helpx.adobe.com/indesign/using/default-keyboard-shortcuts.html)

Why did you choose this data source?
Because it's official site for adobe products.

Are there any other alternative (better) data sources?
No

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
Graphic designers and production artists are the principal users.

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?
No, this not connected to DuckDuckHack Instant Answer.

Are you having any problems? Do you need our help with anything?
No I'm not having any problem.

Where did you hear about DuckDuckHack? (For first time contributors)
Social media sites.

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
screenshot from 2015-12-24 19 36 43
screenshot from 2015-12-24 19 37 19

---
https://duck.co/ia/view/indesign_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@navneetzz Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub ID:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata post coming soon

File: share/goodie/cheat_sheets/json/indesign.json [hunk 1]
```diff
@@ -0,0 +1,1049 @@
+{
+    "id": "indesign_cheat_sheet",
+    "name": "InDesign",
+    "description": "InDesign provides shortcuts to help quickly work in documents without using the mouse",
+    "metadata": {
+        "sourceName": "InDesign Help/Default keyboard shortcuts",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/indesign.json]: @navneetzz Lets change source name to "Adobe"

File: share/goodie/cheat_sheets/json/indesign.json [hunk 2]
```diff
@@ -0,0 +1,1049 @@
+{
+    "id": "indesign_cheat_sheet",
+    "name": "InDesign",
+    "description": "InDesign provides shortcuts to help quickly work in documents without using the mouse",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/indesign.json]: @navneetzz lets cahnge desc to something like "Keyboard shortcuts for Adobe InDesign, a desktop publishing software application"

**Comment by jophab** [comment]
@navneetzz I know you had done a great work for this. :+1: 
But while making cheat sheets make sure that does not  grow very large.
In case of very large contents the cheat sheet should contain only important content among them.
The reason why we are giving source name and url is that If the user want more info they can go there and have it.
So I think we should remove some sections and make it smaller.
Otherwise it will look like a large guide and it will be unpleasant for user who want basic things.

**Comment by navneetzz** [comment]
I'm working on making it small.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=InDesign Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/indesign_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=InDesign Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/indesign_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/indesign.json [hunk 3]
```diff
@@ -0,0 +1,731 @@
...
+                "val": "Go to previous/next validation error"
+            },
+            {
+                "key": "[Ctrl] [Alt] [Shift] [F7]",
+                "val": "Automatically tag text frames and tables"
+            }
+        ],
+        "Indexing": [
+            {
+                "key": "[Shift] [Ctrl] [Alt] [[]",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/indesign.json]: @navneetzz This isn't displaying properly for me - you may want to escape it (and any other braces or brackets that appear in the cheat sheet). There is a section in [the docs](https://duck.co/duckduckhack/goodie_cheat_sheets) (near the bottom) that shows how to do this.

Comment (mintsoft) [inline share/goodie/cheat_sheets/json/indesign.json]: Yeah it should be `[Shift] [Ctrl] [Alt] [\[]` I think

Comment (navneetzz) [inline share/goodie/cheat_sheets/json/indesign.json]: Ya I will do it right away

Comment (navneetzz) [inline share/goodie/cheat_sheets/json/indesign.json]: @mintsoft and @GuiltyDolphin thanks

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=InDesign Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/indesign_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/indesign.json [hunk 4]
```diff
@@ -0,0 +1,731 @@
+{
+    "id": "indesign_cheat_sheet",
+    "name": "InDesign",
+    "description": "Keyboard shortcuts for Adobe InDesign, a desktop publishing software application",
+    "metadata": {
+        "sourceName": "InDesign Help/Default keyboard shortcuts",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/indesign.json]: @navneetzz  Lets change the source name to "Adobe"

Comment (navneetzz) [inline share/goodie/cheat_sheets/json/indesign.json]: Why change name to adobe ?

Comment (navneetzz) [inline share/goodie/cheat_sheets/json/indesign.json]: If you have used it you might know why it is called indesign

**Comment by jophab** [comment]
@navneetzz
The sourcename should reflect the domain of source url.
Here the domain is Adobe.
Thats why I suggested to change that.
If you still have doubts on this, lets do something after getting advice from @moollaza @mintsoft 
:+1:  Happy Holidays :santa:

File: share/goodie/cheat_sheets/json/indesign.json (hunk 4 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/json/indesign.json]: Agreed, let's change the source to `Adobe` -- it’s the domain the user will go to to see more information

**Comment by moollaza** [comment]
@navneetzz @jophab this one is still way too big. Let's strip it down to the essentials. Anything over 300 LOC is a bit concerning. This one is at 700 which is just too much.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=InDesign Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/indesign_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=InDesign Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/indesign_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=InDesign Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/indesign_cheat_sheet_mobile.png)

**Comment by jophab** [comment]
@navneetzz why are you closing this?

**Comment by navneetzz** [comment]
It's no more tracked by original branch and it has collision with my previous submitted cheat sheet
I've created another PR for it.

**Pull request closed without merging**